pkg/ipam/poolallocator/allocator.go

```diff
@@ -608,3 +615,11 @@ func (a *IPPoolAllocator) getReservedIP(reservedOwner v1alpha2.IPAddressOwner) (
 	}
 	return nil, nil
 }
+
+func (a IPPoolAllocator) Total() int {
```

Comment:
just a curiosity: From the code I see this only used in unit tests. did you add this function mostly for checking tesy success?

Reply:
Indeed used only in tests here, but this function is also added in https://github.com/antrea-io/antrea/pull/2995 (we should merge that one).